lib/internal/fs/dir.js

```diff
@@ -4,6 +4,7 @@ const {
   ObjectDefineProperty,
   Symbol,
   SymbolAsyncIterator,
+  PromiseReject
```

Comment:
Please keep the alphabetical order
```suggestion
  ObjectDefineProperty,
  PromiseReject,
  Symbol,
  SymbolAsyncIterator,
```